package.json

```
@@ -30,11 +30,11 @@
     "dist"
   ],
   "packageManagers": {
-    "pnpm": "7.26.1"
+    "pnpm": "8.2.0"
   },
   "engines": {
-    "node": "^14 || ^16 || ^18",
-    "pnpm": "^7.26.1"
+    "node": "^18 || ^19",
```

```suggestion
    "node": "^14.18.0 || >=16.0.0",
```

It is better to align with [Vite](https://github.com/vitejs/vite/blob/ad21ec35de8b6745e30f6982a0e29a1cf97d55c8/package.json#L6).

---

Done.